create/how_to/aws.md

```diff
@@ -0,0 +1,181 @@
+
+Keep in mind that your MeiliSearch instance is currently running in a *development environment*, which is unsafe for production usage. If you want to set up a *production environment*, continue to the [next section](#part-2-configure-production-settings).
+
+Otherwise, if you want to get started creating indexes and adding documents, don't hesitate to check out our [learning resources](/learn), [API Reference](/reference), or [getting started guide](/learn/tutorials/getting_started.md).
+
+And of course, **enjoy**!
+
+## Part 2: Configure production settings
+
+Configuring your MeiliSearch instance in a production environment is not just straightforward—it's completely automated. Simply establish an SSH connection with your instance, and a script will guide you through the process.
+
+### 1. Make your domain name point to your Instance IP
+
+If you want to use your own domain name (or sub-domain), add an `A record` in your domain name provider account. Otherwise, **you can skip this step**.
+
+![Domain to  MeiliSearch](/aws/11.domain.png)
+
+Your domain name should now be linked to your MeiliSearch instance. Run a health check to verify that your instance is running and your DNS is well configured:
+
+```bash
+curl -v http://<your-domain-name>/health
+```
+
+The server should answer with a `204 No content` status code as shown in the example below:
+
+```bash
+...
+< HTTP/1.1 204 No Content
+...
+```
+
+### 2. Set API KEY and SSL (HTTPS)
+
+Meilisearch is currently running in a *development environment*. You haven't set up an **API key**, meaning that anyone can read/write from your MeiliSearch, and you aren't using HTTPS yet, which makes this configuration unsafe for production.
+
+To start the configuration process, connect via SSH to your new MeiliSearch Instance and follow the instructions that appear.
+
+### 2.1. Run the configuration script
+
+Open a terminal and start a new SSH connection with the **Public IPv4** or **your domain name**, using 'admin' as the username.
+
+```bash
+ssh admin@<your-ipv4-address>
+```
+
+```bash
+ssh admin@<your-domain-name>
+```
+
+Write `yes` and press `Enter` to accept the authentication process.
+
+A script will run automatically, asking for your settings and desired configuration. If you want to run this script again at any time, you can do so by using the following command:
+
+```bash
+meilisearch-setup
+```
+
+### 3. Enjoy your ready-to-use MeiliSearch instance
+
+Your MeiliSearch Instance is up and running on AWS, and it is ready to be used in **production**.
+
+To check if everything is running smoothly, do another HTTP call to the `/health` route:
+
+```bash
+curl -v https://<your-domain-name>/health
+```
+
+> Note that this time, we're using HTTPS.
+
+The server should answer with a `204 No content` status code as shown in the example below:
+
+```bash
+...
+< HTTP/1.1 204 No Content
+...
+```
+
+You're all set to use MeiliSearch in production with AWS!
```

Comment:
Is our image in a repo somewhere? If not, should we put it out there so people can contribute to it? No matter about that, we should probably finish with something like: if you have any issues with our AWS image please create issues in the XYZ repo.

Reply:
@eskombro

Reply:
https://github.com/meilisearch/meilisearch-aws

Reply:
Creating an issue into this repo is definitely a good way to help us if there is any trouble 👍

Reply:
Do you mean the AMI itself? If yes, it is stored in our AWS account (for MeilI orga) but the rope that @curquiza linked has all the tools that we use to build it, so it is public and open to contributions

Reply:
Thanks!